CHANGES.rst

```diff
@@ -1,16 +1,18 @@
-Changes
-=======
+=========
+ Changes
+=========
 
 2.2.0 (unreleased)
-------------------
+==================
 
-- Add support for Python 3.5.
+- Add support for Python 3.5 and 3.6.
```

Comment:
`and PyPy3` as this was pending in version 2.1.0.

Reply:
OK.